diff -rupN /media/stor14/Projects/OS/posnk_buildenv/src3p/mpfr-3.1.2/config.sub mpfr-3.1.2/config.sub
--- /media/stor14/Projects/OS/posnk_buildenv/src3p/mpfr-3.1.2/config.sub	2013-03-13 16:38:21.000000000 +0100
+++ mpfr-3.1.2/config.sub	2014-05-07 02:16:57.677145452 +0200
@@ -1348,7 +1348,7 @@ case $os in
 	      | -hpux* | -unos* | -osf* | -luna* | -dgux* | -auroraux* | -solaris* \
 	      | -sym* | -kopensolaris* \
 	      | -amigaos* | -amigados* | -msdos* | -newsos* | -unicos* | -aof* \
-	      | -aos* | -aros* \
+	      | -posnk* | -aos* | -aros* \
 	      | -nindy* | -vxsim* | -vxworks* | -ebmon* | -hms* | -mvs* \
 	      | -clix* | -riscos* | -uniplus* | -iris* | -rtu* | -xenix* \
 	      | -hiux* | -386bsd* | -knetbsd* | -mirbsd* | -netbsd* \
